test/services/samples/group_batch_sample_import_service_test.rb

```diff
@@ -90,30 +61,10 @@ def setup # rubocop:disable Metrics/MethodLength
         content_type: file.content_type
       )
 
-      Samples::BatchFileImportService.new(@project.namespace, @john_doe, blob.id, @default_params).execute
-      assert_equal(@project.namespace.errors.full_messages_for(:base).first,
-                   I18n.t('services.spreadsheet_import.missing_header',
-                          header_title: 'sample_name,project_puid'))
-    end
-
-    test 'import samples into a project that does not belong to project namespace' do
-      assert_equal 3, @project.samples.count
-      assert_equal 20, @project2.samples.count
-
-      response = Samples::BatchFileImportService.new(@project2.namespace, @john_doe, @blob.id,
-                                                     @default_params).execute
-
-      assert_equal 3, @project.samples.count
-      assert_equal 20, @project2.samples.count
-
-      assert_equal I18n.t('services.samples.batch_import.project_puid_not_in_namespace',
-                          project_puid: @project.puid,
-                          namespace: @project2.namespace.full_path),
-                   response['my new sample'][:message]
-      assert_equal I18n.t('services.samples.batch_import.project_puid_not_in_namespace',
-                          project_puid: @project.puid,
-                          namespace: @project2.namespace.full_path),
-                   response['my new sample 2'][:message]
+      Samples::BatchFileImportService.new(@group, @john_doe, blob.id, @default_params).execute
+      assert_equal(I18n.t('services.spreadsheet_import.missing_header',
+                          header_title: 'sample_name,project_puid'),
+                   @group.errors.full_messages_for(:base).first)
```

Comment:
Change this to `@group.errors.full_messages.to_sentence`

Reply:
Fix tests https://github.com/phac-nml/irida-next/pull/992/commits/04f561b5fa2efa5a3f8da68aa5f26e381df149e7